proto/cacheclient.proto

```diff
@@ -1025,3 +1028,35 @@ message _SortedSetLengthByScoreResponse {
     _Missing missing = 2;
   }
 }
+
+message _SortedSetUnionStoreRequest {
+  enum AggregateFunction {
+    SUM = 0;
+    MIN = 1;
+    MAX = 2;
+  }
+
+  message _Source {
+    bytes set_name = 1;
+    optional float weight = 2;
+    AggregateFunction aggregate = 3;
+  }
+
+  bytes set_name = 1;
+  repeated _Source sources = 2;
+  uint64 ttl_milliseconds = 3;
+  bool refresh_ttl = 4;
```

Comment:
I do not think this is going to be supported on the stored set.

Reply:
What is going to be supported? the ttl and refresh ttl capabilities?

Reply:
ttl only

Reply:
Removed refresh_ttl in https://github.com/momentohq/client-protos/pull/310/commits/4d16a5dd8ce4a0cd9e6836374dd600d26d14442c